mir_dataset_loaders/salami.py

```diff
@@ -0,0 +1,210 @@
+
+
+def load(data_home=None, silence=True):
+    validate(data_home, silence)
+    salami_data = {}
+    for key in track_ids():
+        salami_data[key] = load_track(key, data_home=data_home)
+    return salami_data
+
+
+def load_track(track_id, data_home=None):
+    if track_id not in SALAMI_INDEX.keys():
+        raise ValueError(
+            "{} is not a valid track ID in Salami".format(track_id))
+
+    if SALAMI_METADATA is None or SALAMI_METADATA['data_home'] != data_home:
+        _reload_metadata(data_home)
+        if SALAMI_METADATA is None:
+            raise EnvironmentError("Could not find Salami metadata file")
+
+    if track_id in SALAMI_METADATA.keys():
+        track_metadata = SALAMI_METADATA[track_id]
+    else:
+        # annotations with missing metadata
+        track_metadata = {'source': None, 'annotator_1_id': None, 'annotator_2_id': None, 'duration_sec': None,
+                          'title': None, 'artist': None, 'annotator_1_time': None, 'annotator_2_time': None,
+                          'class': None, 'genre': None}
+
+    annotations_dir = os.path.join(data_home, SALAMI_DIR, 'salami-data-public-master', 'annotations')
+    annotators = [any(SALAMI_INDEX[track_id]['annotator_1_uppercase']),
+                  any(SALAMI_INDEX[track_id]['annotator_2_uppercase'])]
+    all_annotators_section_data = _load_sections(get_local_path(annotations_dir, track_id), annotators)
+
+    return SalamiTrack(
+        track_id,
+        all_annotators_section_data[0],
+        all_annotators_section_data[1],
+        all_annotators_section_data[2],
+        all_annotators_section_data[3],
+        track_metadata['source'],
+        track_metadata['annotator_1_id'],
+        track_metadata['annotator_2_id'],
+        track_metadata['duration_sec'],
+        track_metadata['title'],
+        track_metadata['artist'],
+        track_metadata['annotator_1_time'],
+        track_metadata['annotator_2_time'],
+        track_metadata['class'],
+        track_metadata['genre']
+    )
+
+
+def _load_sections(sections_path, annotators):
+    all_annotators_section_data = []
+    for a in range(len(annotators)):
+        for f in ['uppercase.txt', 'lowercase.txt']:
+            times, secs = [], []
+            if annotators[a]:
+                file_path = os.path.join(sections_path, 'parsed',
+                                         'textfile{}_{}'.format(str(a + 1), f))
+                if os.path.exists(file_path):
+
+                    with open(file_path, 'r') as fhandle:
+                            reader = csv.reader(fhandle, delimiter='\t')
+                            for line in reader:
+                                times.append(float(line[0]))
+                                secs.append(line[1])
+                    times, secs = np.array(times), np.array(secs)
+                    # remove sections with length == 0
+                    times_ = np.delete(times, np.where(np.diff(times) == 0))
```

Comment:
I think naming a variable with a trailing underscore means something specific in python, at least by convention. How about instead:
`times_` --> `times_revised`
and for consistency do the same with `secs`
`secs_revised`